src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -5,7 +5,9 @@
 using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text.RegularExpressions;
 using System.Threading;
+using Microsoft.Graph;
```

Comment:
remove this?

Reply:
my IDE will automatically add some strange namespace... :(

Reply:
me too, it is a new annoying IDE feature